services/skus/service.go

```diff
@@ -2232,6 +2254,126 @@ func (s *Service) processSubmitReceipt(ctx context.Context, req model.ReceiptReq
 	return rcpt, nil
 }
 
+const errUnknownAction = model.Error("skus: unknown radom action")
+
+func (s *Service) processRadomEvent(ctx context.Context, event *radom.Event) error {
+	if !event.ShouldProcess() {
+		return nil
+	}
+
+	tx, err := s.Datastore.RawDB().BeginTxx(ctx, nil)
+	if err != nil {
+		return err
+	}
+	defer func() { _ = tx.Rollback() }()
+
+	if err := s.processRadomEventTx(ctx, tx, event); err != nil {
+		return err
+	}
+
+	return tx.Commit()
+}
+
+func (s *Service) processRadomEventTx(ctx context.Context, dbi sqlx.ExtContext, event *radom.Event) error {
+	switch {
+	case event.IsNewSub():
+		oid, err := event.OrderID()
+		if err != nil {
+			return err
+		}
+
+		subID, err := event.SubID()
+		if err != nil {
+			return err
+		}
+
+		rsub, err := s.radomClient.GetSubscription(ctx, subID)
+		if err != nil {
+			return err
+		}
+
+		nxtB, err := rsub.NextBillingDate()
+		if err != nil {
+			return err
+		}
+
+		expAt := nxtB.Add(24 * time.Hour)
```

Comment:
suggestion (blocking): `renewOrderWithExpPaidTimeTx` will add 24 hours. I assume this code had been written right around the time I made that change (to unify this logic in one place). So this addition is no longer necessary, as it already happens in `master` in `renewOrderWithExpPaidTimeTx`.

Reply:
I dont see that in the `master` on the `renewOrderWithExpPaidTimeTx` [method](https://github.com/brave-intl/bat-go/blob/3265249ff70a63fd66922ba85639e1be2156750d/services/skus/service.go#L2018). 

24 hours still seems to be added in other places e.g. [play](https://github.com/brave-intl/bat-go/blob/3265249ff70a63fd66922ba85639e1be2156750d/services/skus/service.go#L1752) and [apple](https://github.com/brave-intl/bat-go/blob/3265249ff70a63fd66922ba85639e1be2156750d/services/skus/service.go#L1702) which are calling the same method.

Reply:
Oh sorry, I have confused it. It's the code for Stripe has now been unified, and somehow I thought about it. Irrelevant, resolving.